### PR TITLE
VSPIX not supported on ARM64 VS

### PR DESCRIPTION
**Pull request by davidcongruili**
Adding clarification that ARM64 does not support VSPIX

### CODE REVIEW

**Comment by jborsecnik** [comment]
@Mikejo5000, 

Can you review the proposed changes? Note: The first changed needs a line break added (apparently) and a period.

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/debugger/graphics/visual-studio-graphics-diagnostics.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ ms.workload:
  [!INCLUDE [Visual Studio](~/includes/applies-to-version/vs-windows-only.md)]
 >[!NOTE]
 > Visual Studio recommends PIX on Windows for DirectX 12 games. [PIX on Windows](https://aka.ms/PIXonWindows) is a performance tuning and debugging tool that fully supports DirectX 12. [Find out more information](visual-studio-graphics-diagnostics-directx-12.md) or [download here](https://aka.ms/downloadPIX).
+> Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio
```
Comment (Mikejo5000) [inline docs/debugger/graphics/visual-studio-graphics-diagnostics.md:22]:
```suggestion
> Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio.
```

File: docs/profiling/gpu-usage.md [hunk 2]
```diff
@@ -13,7 +13,8 @@ ms.workload:
 # GPU usage
 
  [!INCLUDE [Visual Studio](~/includes/applies-to-version/vs-windows-only.md)]
-
+>[!NOTE]
+> Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio
```
Comment (Mikejo5000) [inline docs/profiling/gpu-usage.md:17]:
```suggestion
> Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio.
```

File: docs/profiling/gpu-usage.md [hunk 3]
```diff
@@ -13,7 +13,8 @@ ms.workload:
...
 Use the GPU Usage tool in the Performance Profiler to better understand the high-level hardware usage of your Direct3D app. It helps you see whether the performance of your app is CPU-bound or GPU-bound, and gain insight into how you can use the platform's hardware more effectively. GPU Usage supports apps that use Direct3D 12, Direct3D 11, and Direct3D 10. It doesn't support other graphics APIs, such as Direct2D or OpenGL.
```
Comment (Mikejo5000) [inline docs/profiling/gpu-usage.md:18]:
```suggestion

Use the GPU Usage tool in the Performance Profiler to better understand the high-level hardware usage of your Direct3D app. It helps you see whether the performance of your app is CPU-bound or GPU-bound, and gain insight into how you can use the platform's hardware more effectively. GPU Usage supports apps that use Direct3D 12, Direct3D 11, and Direct3D 10. It doesn't support other graphics APIs, such as Direct2D or OpenGL.
```

**Comment by Mikejo5000** [comment]
#sign-off

File: docs/profiling/gpu-usage.md [hunk 4]
```diff
@@ -13,6 +13,8 @@ ms.workload:
 # GPU usage
 
  [!INCLUDE [Visual Studio](~/includes/applies-to-version/vs-windows-only.md)]
+>[!NOTE]
```
Comment (Jak-MS) [inline docs/profiling/gpu-usage.md:16]:
```suggestion

>[!NOTE]
```

File: docs/profiling/gpu-usage.md [hunk 5]
```diff
@@ -13,6 +13,8 @@ ms.workload:
...
+> Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio.
```
Comment (Jak-MS) [inline docs/profiling/gpu-usage.md:18]:
```suggestion

```
removing this duplication

File: docs/profiling/gpu-usage.md [hunk 6]
```diff
@@ -13,6 +13,8 @@ ms.workload:
...
 
 >[!NOTE]
 > Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio
```
Comment (Jak-MS) [inline docs/profiling/gpu-usage.md:20]:
```suggestion
> Visual Studio Graphics Diagnostics is not supported on ARM64 Visual Studio.
```

File: docs/profiling/gpu-usage.md (hunk 5 above)
Comment (Jak-MS) [inline docs/profiling/gpu-usage.md:18]:
```suggestion

```

**Pull request merged**